n3fit/src/n3fit/backends/keras_backend/callbacks.py

```diff
@@ -15,9 +15,11 @@
 import logging
 from time import time
 
+# Callbacks need tensorflow installed even if the backend is pytorch
```

Comment:
Really? :| 

It seems keras defaults to the tf backend so there should either be a toggle to set the backend from the runcard and/or and explanation in the docs saying to do `KERAS_BACKEND=torch n3fit`

Reply:
This is because of tensorboard. But I'll just wrap it with try-except